doc/nrf/releases/release-notes-1.0.0.rst

```diff
@@ -112,11 +112,11 @@ nRF9160
 * nRF Connect SDK now uses upstream CoAP implementation. The :ref:`mqtt_simple_sample` sample was rewritten to use the upstream library, and the downstream CoAP was removed.
 * The :ref:`http_application_update_sample` sample has been updated to use the :ref:`lib_fota_download` library.
 
-Modem library
+BSD library
 -------------
```

Comment:
I suppose these should stop the the same column as `BSD library`

Reply:
Fixed these..